bool imageStatus
bool odomStatus
bool orbStartStatus
bool orbInitStatus
bool orbScaleStatus
bool orbGCFlag
bool orbGBAFlag
bool chargingStatus
uint32 brightness
float64 power
